### ISSUE TITLE
Test DSD 

### ISSUE THREAD

**Issue by razekmh** [SAFEHR-data/ramses-package#41]
I want to test using the UCLH Data Science Desktop for R development:

- [x] Which version of R works best with the limitations of DSD?
- [x] Which version of R studio?
- [x] Can I use VS code for development?
- [x] Can I clone from GitHub?
- [x] Can I push to GitHub? 
- [x] How to clone large files from GitHub to the DSD?

**Comment by razekmh** [comment]
**Which version of R works best with the limitations of DSD?**

4.1.3

It is not mentioned explicitly in the slab docs (AFAIK) but I tried most recent R version and they broke the R Studio. When installing the latest R version `4.4.2` and opening R studio, the program opens for a second and then crashes. While using the version of R the DSD comes with `4.0.0` does not support many of the basic functions of tidyverse.

**Comment by razekmh** [comment]
**Which version of R studio?**

I tried installing newer versions but I did not have enough permissions. I guess we are using the one that comes with the DSD `1.2.5042`

**Comment by razekmh** [comment]
**Can I use VS code for development?**

Partially, I can use VS code to edit files but I cannot run R code inside it. I tried installing the R integrating extension but I did not have enough permissions

**Comment by razekmh** [comment]
**Can I clone from GitHub?**

Yes, but I have to update the proxy settings ref to [the guide on Slab](https://uclh.slab.com/posts/trouble-shooting-guide-for-developers-5xzqj5h3)

```
git config --global http.proxy http://www-cache-n.xuclh.nhs.uk:3128    
git config --global https.proxy https://www-cache-n.xuclh.nhs.uk:3128

**Comment by razekmh** [comment]
**Can I push to GitHub?**
Yes, but only using the bash CLI. I have not been able to push using the integration of VS so far. VS code cannot pull the most recent updates from GitHub. I think I might need to edit some of its config for this. 

Edit: you can edit the config to use the proxy. You will need to sign in to your GitHub account too
```
{
    "http.proxy": "https://www-cache-n.xuclh.nhs.uk:3128"
}
```

**Comment by razekmh** [comment]
**How to clone large files from GitHub to the DSD?**

You cannot. Although `git-lfs/2.10.0` is installed. I cannot clone the [vocab repo](https://github.com/SAFEHR-data/omop-vocabs-processed) because the request times out. Maybe it is something that has to do with the proxy? 

I tried downloading the zip file from GitHub but that did not work either for the same reason. The request times out! 

I tried opening each individual file on GitHub and downloading the raw file directly. This one worked, so I downloaded each file. I understand that this means I will need to update the files manually but it works for now.
